client/components/vertical-menu/style.scss

```diff
@@ -22,8 +22,12 @@
 
 	&.is-selected {
 		border-left: 3px solid $blue-wordpress;
-		color: darken( $gray, 30% );
+		color: $gray-dark;
```

Comment:
Looking at #11368, eventually `$gray-text: $gray-dark;`, so there won't be any difference anymore between here and the normal state of the menu items (line 14).

Maybe it would make more sense to use `$gray-text-min` (which is lighter) for the normal state and `$gray-text` for the selected one?